src/models/eventmodel.cpp

```diff
@@ -250,6 +259,9 @@ QVariant EventModel::data(const QModelIndex& index, int role) const
         Q_ASSERT(thread);
     } else if (tag == Tag::Tracepoints) {
         tracepoint = &m_data.tracepoints[index.row()];
+    } else if (tag == Tag::Favorites) {
+        auto& favourite = m_favourites[index.row()];
```

Comment:
const, or just inline directly into the `data()` call below

Reply:
this will be wrong when we get a request for column 1 but the favorite is for column  0 or vice versa